### PR TITLE
Mavaca/arduino review

### PR DESCRIPTION
**Pull request by maye-vaca**
Update activity names
Update instructions
Update wiring diagrams
Include fritzing files, powerpoint for future updates

### CODE REVIEW

File: content/english/arduino/Circuit-Basics.md [hunk 1]
```diff
@@ -1,35 +1,34 @@
...
-+ The Anode is longer than the Cathode and is always attached to the higher voltage.
-![Cannot load image](../img/img18.png)
-
-The resistor is a small piece of ceramic material that slows down the flow of electricity. 
-
-+ A resistor has 2 leads, but it is omnidirectional. In other words, it doesn’t matter which lead you attach to the higher voltage. 
-+ Most electronics can only handle a certain amount of electricity before they break, so resistors protect the other components in our game by keeping the electricity low. 
-![Cannot load image](../img/img19.png)
+---
+title: "Circuit Basics"
```
Comment (ozhang22) [inline content/english/arduino/Circuit-Basics.md:2]: Can you name the file in lowercase?

File: content/english/arduino/Circuit-Basics.md [hunk 2]
```diff
@@ -1,35 +1,34 @@
...
+description: ""
+draft: false
+hidden: false
+weight: 3
+---
+
+# Appendix
+
+## The components, in detail
```
Comment (ozhang22) [inline content/english/arduino/Circuit-Basics.md:11]: This is not the appendix anymore.

```suggestion
## The components, in detail
```

File: content/english/arduino/Circuit-Basics.md [hunk 3]
```diff
@@ -1,35 +1,34 @@
...
+Imagine our circuit is a play. Before the final performance in front of an audience, we need a place to practice. This is where the breadboard comes in. 
++ Breadboards are simple platforms that we can use to build temporary circuits for testing and debugging before finalizing our design and permanently soldering (welding) it together. 
++ Breadboards allow us to easily add, remove, or reposition components. Below on the left is an image of a breadboard.
++ The image to the right shows how the pins are connected to each other. 
++ Each one of the numbered rows on the breadboard, shown in yellow, contains 5 pins holes. Every pin hole on a row (yellow line) is connected to the other pin holes on that line. 
++ The red line on the right shows a common connection between those pin holes, while the blue line on the right side show a common connection between those pin holes. 
++ The same is true on the red and blue lines on the left. However, the red and blue lines on the right and left are not connected to each other.
++ Components that are plugged into the same common connection are connected in the circuit
+
+![Cannot load image](../img/img17.png)
```
Comment (ozhang22) [inline content/english/arduino/Circuit-Basics.md:22]: Some notes about images:

- Can you change all the [Cannot load image] alt text into an appropriate description of the image?
- Renaming the images to match what they are talking about would be nice too
- Do the images look OK? Might have to use HTML and resize the images to an appropriate size.

File: content/english/arduino/activity-2.md [hunk 4]
```diff
@@ -1,20 +1,25 @@
...
-5.	Attach one end of a jumper wire to SW on the joystick and the other to 2 on the Elegoo.
+3.	Attach one end of a jumper wire to VRx on the joystick and the other to A0 (analog pin 0) on the Elegoo.
+4.	Attach one end of a jumper wire to VRy on the joystick and the other to A1 (analog pin 1) on the Elegoo.
+4.	Attach one end of a jumper wire to VRy on the joystick and the other to A1 (analog pin 1) on the Elegoo.
+5.	Attach one end of a jumper wire to SW on the joystick and the other to 2 (digital pin 2) on the Elegoo.
 6.	Plug in the Arduino board via USB to your PC and open the Arduino IDE application.
 
-CODE:
+### Recognizing and printing Analog Inputs
+We will again use sample code for our programs. This time from the elegoo website itself. Copy the following into your Arduino IDE.
```
Comment (ozhang22) [inline content/english/arduino/activity-2.md:22]:
```suggestion
We will again use sample code for our programs. This time we'll get it from the elegoo website itself. Copy the following into your Arduino IDE.

```

File: content/english/arduino/setting-up-the-pc.md [hunk 5]
```diff
@@ -1,15 +1,15 @@
...
+weight: 1
 ---
 
-# Setting up the PC
+## Getting to know the Arduino IDE
 
 Download and install the Arduino IDE from:
 * the Microsoft Store (quicker install) - https://www.microsoft.com/store/productId/9NBLGGH4RSD8
-* or from the Arduino website https://www.arduino.cc/en/Main/Software (current version is 1.8.9)
+* or from the Arduino website https://www.arduino.cc/en/Main/Software (current version is 1.8.13)
```
Comment (ozhang22) [inline content/english/arduino/setting-up-the-pc.md:12]: I don't like that we mention the current version - it means we need to keep updating this. I would just say:

```suggestion
* or from the Arduino website https://www.arduino.cc/en/Main/Software (pick the latest version)
```

File: content/english/arduino/activity-4.md [hunk 6]
```diff
@@ -1,16 +1,17 @@
 ---
-title: "Arduino - Activity 4"
+title: "Activity 4 - LCD Discplay"
```
Comment (ozhang22) [inline content/english/arduino/activity-4.md:2]:
```suggestion
title: "Activity 4 - LCD Display"

```

File: content/english/arduino/_index.md [hunk 7]
```diff
@@ -5,9 +5,18 @@ date: 2020-07-27T14:04:59.61-07:00
...
 
+## Introduction - Building with Arduino
+
+Electronics like microcontrollers, sensors, and all types of circuitry are all around us! Some are very basic and provide telemetry information for traffic systems. Others run the traffic lights programmed with built-in timing.
+
+Today we will work on a few activities to show you some of the capabilities of this platform. We will start small and simple and build to bigger and more complex. You will learn all the basic information about how to use the Arduino controller board, sensors, and components. After you do each activity, feel free to change some things to see what is different and gain a deeper understanding.
+
+![Picture of an Arduino](./img/img1.png)
+
+
```
Comment (ozhang22) [inline content/english/arduino/_index.md:19]:
```suggestion
```

File: content/english/arduino/activity-1.md [hunk 8]
```diff
@@ -1,29 +1,37 @@
...
-title: "Arduino - Activity 1"
+title: "Activity 1: Blinking LED"
 draft: false
-hidden: true
+hidden: false
 weight: 4
 ---
 
-### Activity 1 - Blinking LED
+### Wiring Our Blinking LED
```
Comment (ozhang22) [inline content/english/arduino/activity-1.md:8]: Inconsistency with using ### vs ## for these subheaders - can you pick one?

**Review by ozhang22 — CHANGES_REQUESTED** [review]
Have some comments - please take a look @maye-vaca.

File: content/english/arduino/Circuit-Basics.md (hunk 3 above)
Comment (maye-vaca) [reply content/english/arduino/Circuit-Basics.md]: I think they look okay but I did rename the ones that are in use.

**Review by ozhang22 — APPROVED** [review]

**Pull request merged**